docs/built-in-models/neural-networks.md

```diff
@@ -36,11 +33,6 @@ n_inputs = 10
 n_outputs = 2
 params = {
     "module__n_layers": 2,
-    "module__n_w_bits": 2,
```

Comment:
why the removal ?

Reply:
if default parameters are good, it's best to give examples with default parameters -> it's best to showcase ease-of-use instead of complexity

Reply:
yeah indeed !